erpnext/payroll/report/ecs_checklist/ecs_checklist.json

```diff
@@ -0,0 +1,30 @@
+{
+ "add_total_row": 0,
+ "creation": "2020-06-01 18:18:54.406461",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "modified": "2020-06-01 18:18:54.406461",
+ "modified_by": "Administrator",
+ "module": "Payroll",
+ "name": "ECS Checklist",
```

Comment:
Rename the report, may be "Salary Payments via ECS"

Reply:
done